packages/fluentui/e2e/tests/dialogPreventScroll-example.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { Button, Dialog } from '@fluentui/react-northstar';
+
+export const selectors = {
+  outerClose: 'outer-close',
```

Comment:
Do we need all these selectors for this test?

Reply:
Removed